### PR TITLE
Feature/sdl 0264 separating the change of audible status and the change of hmi status

### PR DESCRIPTION
**Pull request by Yuki-Kanadome-Nexty**
Implements/Fixes [#275 ](https://github.com/smartdevicelink/sdl_hmi/issues/275)

This PR is **ready** for review.

### Testing Plan
This change will be tested with sdl_core 7.0 + PR at [smartdevicelink/sdl_core#3634](https://github.com/smartdevicelink/sdl_core/pull/3634).

### Summary
This is a response to changes made by SDL 0264.
When sending OnEventChanged (PHONE_CALL, true / false), OnAppDeactivated / OnAppActivated is sent to the app which HMI Level is FULL.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Review by jacobkeeler — CHANGES_REQUESTED** [review]
While there are a few changes I would suggest, after looking more into this I don't think this PR is needed. The original behavior of the HMI was to stay on the phone screen after the phone call was ended, so I don't see any reason to add this behavior now.

**Comment by jordynmackool** [comment]
@Yuki-Kanadome-Nexty please see Jacobs's feedback above and let us know your thoughts. If you agree with Jacob, we can then close this PR and the associated issue.

**Comment by jordynmackool** [comment]
@Yuki-Kanadome-Nexty Following up to ensure you saw Jacobs comment [here](https://github.com/smartdevicelink/sdl_hmi/pull/499#pullrequestreview-691873436). If we don't hear from you this week, we will move forward with closing the Issue/ PR on 2021-07-12.

**Comment by Akihiro-Miyazaki** [comment]
@jacobkeeler -san,
Sorry for delay our response.
Does your suggestion mean below?
 Ex. Music screen - (phone call) -> Phone call screen - (phone call is ends) -> Stay phone call screen
If our recognition is wrong, could you please explain the details.
Our proposal is to return to the screen before phone call like below.
 Ex. Music screen - (phone call) -> Phone call screen - (phone call is ends) -> Music screen
Therefore, we made a commit the modification.

**Comment by jacobkeeler** [comment]
Ah, I can see how this case would make sense. It does get a bit more complicated if you leave the phone screen before ending the call though:

Music app screen - (phone call) -> Phone call screen -> App list - (activate nav app) -> Navigation app screen -> Phone call screen - (phone call ends) -> Music app screen?

Perhaps we should clear `redisplayAppID` if the user leaves the phone call screen?

File: app/model/sdl/Abstract/data.js [hunk 1]
```diff
@@ -1574,6 +1574,11 @@ SDL.SDLModelData = Em.Object.create(
       'UNAUTHORIZED_TRANSPORT_REGISTRATION': -3,
       'RESOURCE_CONSTRAINT': -4,
       'CLOSE_CLOUD_CONNECTION': -5
-    }
+    },
+    
+    /**
+     * App ID to be redisplayed after the Phone call ends
+     */
+    redisplayAppID: null,
```
Comment (jacobkeeler) [inline app/model/sdl/Abstract/data.js:1582]: Perhaps this should be included in `PhoneModel.js` instead?

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by jacobkeeler** [comment]
Closing as review has moved to #576

**Pull request closed without merging**